db_stress_tool/db_stress_driver.cc

```diff
@@ -30,8 +31,9 @@ void ThreadBody(void* v) {
       shared->GetCondVar()->Wait();
     }
   }
-  thread->shared->GetStressTest()->OperateDb(thread);
-
+  if (!FLAGS_post_verification_only) {
+    thread->shared->GetStressTest()->OperateDb(thread);
+  }
```

Comment:
Sorry - I just started to catch up on this. 

My learning question: if we skip db operation in FLAGS_post_verification_only=true as quoted, then  is this "post verification" essentially a "pre-run verification" in the uninterrupted final run? 

The reason for this question is that I'm trying to understand why we don't have pre_verification_only instead though the implementation is pretty much the same, except that it seems slightly easier to keep [FLAGS_preserve_unverified_changes](https://github.com/facebook/rocksdb/blob/main/db_stress_tool/db_stress_driver.cc#L62C1-L62C1) working with this type of verification-only run.